drivers/isl29125/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = isl29125
```

Comment:
I think automatically setting of the MODULE name was introduced somewhere on the way, so many prio drivers to not use that feature (but no reason not to use it for new drivers...). Could be a good newbie-task-candidate to unify them all?!